index.html

```diff
@@ -726,6 +721,19 @@ <h4 id="query-method">
             <var>permissionDesc</var>:
           </p>
           <ol class="algorithm">
+            <li>If |permissionDesc|'s {{PermissionDescriptor/name}} member is not supported, return
+            [=a promise rejected with=] a {{TypeError}}.
+              <aside class="note" title="Why is this not an enum?">
+                <p>
+                  This is deliberately design to work the same a WebIDL [=enumeration=] (`enum`)
+                  and implementers are encouraged to use their own custom `enum` here. The reason
+                  this is not an enum in the specification vary greatly in the powerful features
+                  they support. Using a {{DOMString}} to identify a powerful feature gives
+                  implementers the freedom to pick and choose which of the power features from the
+                  [[[powerful-feature-registry]]] they wish to support.
```

Comment:
This links out to https://w3c.github.io/poweful-features-registry/ (note the typo in powerful) - presumably we fix that elsewhere.

Reply:
Fixed elsewhere (main) and rebased... now made it so once we merge this, it should link without needing the localBiblio.